app/views/jobs/_ganglia_grpah_table.html.erb

```diff
@@ -0,0 +1,28 @@
+<table>
+<div class="panel panel-default">
+	<tr class="col-xs-12">
+		<div class="col-xs-12">
+   			<tr><div class="job_data" data-jobid="<%= d.pbsid %>" ></div></tr>
+  		</div>
+    </tr>
+	<% if !d.nodes.nil? && has_ganglia(d.cluster) %>
+		<% d.nodes.each do |node| %>
+			<div class="col-md-12">
+				<tr>
+					<td>
+						<div class="panel panel-default">
+							<div class="panel-heading"><h3 class="panel-title"> <%= node %> <span class="pull-right"> <%= d.pbsid %> </span></h3></div>
+							<div class="panel-body">
+								<a href=" <%= build_ganglia_link(d.cluster, d.starttime, 'cpu_report', node, 'large') %>" data-lightbox="cpu-report" data-title="CPU Report <%= node %>"><img class="img-responsive col-lg-3 col-md-3 col-sm-6 col-xs-6" src="<%= build_ganglia_link(d.cluster, d.starttime, 'cpu_report', node, 'small') %>" /></a>
+								<a href="<%= build_ganglia_link(d.cluster, d.starttime, 'load_report', node, 'large') %>"data-lightbox="load-report" data-title="Load Report <%= node %>"><img class="img-responsive col-lg-3 col-md-3 col-sm-6 col-xs-6" src="<%= build_ganglia_link(d.cluster, d.starttime, 'load_report', node, 'small') %>"/></a>
+								<a href="<%= build_ganglia_link(d.cluster, d.starttime, 'mem_report', node, 'large') %>" data-lightbox="mem-report" data-title="Memory Report <%= node %>"><img class="img-responsive col-lg-3 col-md-3 col-sm-6 col-xs-6" src="<%= build_ganglia_link(d.cluster, d.starttime, 'mem_report', node, 'small') %>" /></a>
+								<a href="<%= build_ganglia_link(d.cluster, d.starttime, 'network_report', node, 'large') %>" data-lightbox="network-report" data-title="Network Report <%= node %>"><img class="img-responsive col-lg-3 col-md-3 col-sm-6 col-xs-6" src="<%= build_ganglia_link(d.cluster, d.starttime, 'network_report', node, 'small') %>"/></a>

```

Comment:
Since this is erb now, we can use https://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to with the do block so you can put the image inside, and you can use image_tag helper there as well